### ISSUE TITLE
Please add support for Composer 2.x

### ISSUE THREAD

**Issue by dmanva** [joshbrw/laravel-module-installer#7]
Next error occurs when attempting to install with composer 2

>   Problem 1
    - joshbrw/laravel-module-installer is locked to version v0.1.4 and an update of this package was not requested.
    - joshbrw/laravel-module-installer v0.1.4 requires composer-plugin-api ^1.1 -> found composer-plugin-api[2.0.0] but it does not match the constraint.

> You are using Composer 2, which some of your plugins seem to be incompatible with. Make sure you update your plugins or report a plugin-issue to ask them to support Composer 2.

**Comment by dyanakiev** [comment]
@dmanva Hm i wonder if this package is even required.

I think this can be done by composer itself?
https://getcomposer.org/doc/faqs/how-do-i-install-a-package-to-a-custom-path-for-my-framework.md
```
{
    "extra": {
        "installer-paths": {
            "Modules/{$name}": ["type:laravel-module"]
        }
    }
}
```

**Comment by IamJustAKid** [comment]
I am having the same exception.

**Comment by darkperis** [comment]
I have the same issue. Please support composer 2.x.

**Comment by joshbrw** [comment]
Hi all.

I'm implementing this now.

**Comment by joshbrw** [comment]
This has now been implemented in v1.0.1. You can upgrade by:

1. Run `composer self-update --2` to upgrade to Composer version 2.
2. Change the `joshbrw/laravel-module-installer` constraint to `^1.0`. 
3. Run `composer update`.
